src/data/automation.ts

```diff
@@ -101,7 +101,7 @@ export interface TagTrigger {
 
 export interface TimeTrigger {
   platform: "time";
-  at: string;
+  at: string[];
```

Comment:
This is not possible? It can't suddenly change from string to an array? Should this be `string | string[]`?